.github/workflows/test.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - develop
+    - visualizer
```

Comment:
This doesn't really belong here. To get CI to run while you're still developing, you can just make a draft PR. Note that you can continue to commit to both draft and non-draft PR branches after they're opened to resolve issues, you don't need to close and then reopen later.

Reply:
I have not checked the code; trust that that is ok.
Have only read the readme.
I'm glad that we reached this milestone and propose to continue with the merge.

Reply:
Removed this line. Also removed some other personal development lines from .gitignore.